src/pages/myNotes/myNotes.jsx

```diff
@@ -3,24 +3,39 @@ import PropTypes from 'prop-types';
 import { Box } from '@material-ui/core';
 import Note from './note';
 import NoteContent from './noteContent';
-import useStyles from './styled';
+import useStyles from '../../styled';
+import Header from '../../shared/header';
+import EditNotesPanel from './editNotesPanel/editNotesPanel';
 
-const MyNotes = ({ notActiveNote, changeNoteContent, initialNote, editPanel, openEditPannel, notesList, onChange, onUpdate, onCancel }) => {
+const MyNotes = ({ notActiveNote, changeNoteContent, initialNote, editPanel, openEditPannel, notesList, onChange, onUpdate, onCancel, openNavBar }) => {
   const styles = useStyles();
   return (
-    <Box className={styles.container}>
-      <Box>
-        {notesList.map((note) => <Note notActive={notActiveNote} changeNoteContent={changeNoteContent} activeNote={note} key={note.id} />)}
+    <>
+      <Header openNavBar={openNavBar} />
+      <Box className={styles.container}>
+        <Box>
+          {notesList.map((note) => <Note notActive={notActiveNote} changeNoteContent={changeNoteContent} activeNote={note} key={note.id} />)}
+        </Box>
+        <Box>
+          <NoteContent
+            initialNote={initialNote}
+            editPanel={editPanel}
+            openEditPannel={openEditPannel}
+            onChange={onChange}
+            onUpdate={onUpdate}
+            onCancel={onCancel}
+          />
+          {editPanel === false ? null : <EditNotesPanel
```

Comment:
There is no need to check by comparing with `false` value and there is no need to render `null`. In such cases use please:
`{editPanel && <EditNotesPanel />}`